src/settings/components/ExternalDataSourcesConfig/ExternalDataSourcesFields/ExternalDataSourcesFields.test.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+
+import { renderWithIntl } from '@folio/stripes-erm-testing';
+import { MemoryRouter } from 'react-router-dom';
+
+import translationsProperties from '../../../../../test/helpers';
+import { input, mutators } from './testResources';
+import ExternalDataSourcesFields from './ExternalDataSourcesFields';
+
+const onDeleteMock = jest.fn();
+const onSaveMock = jest.fn();
+
+jest.mock('../ExternalDataSourcesView', () => () => <div>ExternalDataSourcesView</div>);
+
+let renderComponent;
+
+describe('ExternalDataSourcesFields', () => {
+  beforeEach(() => {
+    renderComponent = renderWithIntl(
+      <MemoryRouter>
+        <ExternalDataSourcesFields
+          input={input}
```

Comment:
We should never send `input` or `meta` props in directly unless there's a very good reason to.